tutordiscovery/plugin.py

```diff
@@ -32,6 +32,8 @@
         "OAUTH2_KEY_SSO_DEV": "discovery-sso-dev",
         "CACHE_REDIS_DB": "{{ OPENEDX_CACHE_REDIS_DB }}",
         "EXTRA_PIP_REQUIREMENTS": [],
+        "ATLAS_PULL": False,
+        "ATLAS_ARGS": "",
```

Comment:
Don't add these configuration settings. Assume all users should run Atlas by default. Why wouldn't they?

Reply:
> Don't add these configuration settings. Assume all users should run Atlas by default. Why wouldn't they?

I understand the argument that `ATLAS_PULL` can be removed and enabled by default.

However, suppose a engineer wants to select which languages atlas will pull how should we go about that?
For example, we need `atlas pull` to have the `--filter=ar,fr_CA` argument on my build. Without  `ATLAS_ARGS`, I'd need to fork the `tutor-discovery` plugin and edit the Dockerfile or maybe add a copy of the Dockerfile template in my local configs.

I'm assuming we could use a global tutor-wide `ATLAS_ARGS` if this helps to reduce the need for forking.
We could alternatively use a `patch` placeholder for extensibility:

`atlas pull {{ patch("atlas-extra-args") }} ...`

The full atlas docs are available here: https://github.com/openedx/openedx-atlas#usage

What do you think?